hermes-console-vue/src/views/topic/TopicView.vue

```diff
@@ -0,0 +1,86 @@
+<script async setup lang="ts">
+  import { useI18n } from 'vue-i18n';
+  import { useRoute } from 'vue-router';
+  import { useSubscriptionsList } from '@/composables/topic-subscriptions/use-subscriptions-list/useSubscriptionsList';
+  import { useTopic } from '@/composables/topic/use-topic/useTopic';
+  import { useTopicMessagesPreview } from '@/composables/topic/use-topic-messages-preview/useTopicMessagesPreview';
+  import { useTopicMetrics } from '@/composables/topic/use-topic-metric/useTopicMetrics';
+  import ConsoleAlert from '@/components/console-alert/ConsoleAlert.vue';
+  import LoadingSpinner from '@/components/loading-spinner/LoadingSpinner.vue';
+  import MessagesPreview from '@/views/topic/messages-preview/MessagesPreview.vue';
+  import MetricsList from '@/views/topic/metrics-list/MetricsList.vue';
+  import PropertiesList from '@/views/topic/properties-list/PropertiesList.vue';
+  import SchemaPanel from '@/views/topic/schema-panel/SchemaPanel.vue';
+  import SubscriptionsList from '@/views/topic/subscriptions-list/SubscriptionsList.vue';
+  import TopicHeader from '@/views/topic/topic-header/TopicHeader.vue';
+
+  const { t } = useI18n();
+  const route = useRoute();
+  const { groupId, topicName } = route.params as Record<string, string>;
+  const { topic, owner, topicIsLoading, ownerIsLoading, topicError } =
+    useTopic(topicName);
+  const { subscriptions } = useSubscriptionsList(topicName);
+  const { data: metrics } = useTopicMetrics(topicName);
+  const { data: messages } = useTopicMessagesPreview(topicName);
+
+  const breadcrumbsItems = [
+    {
+      title: t('subscription.subscriptionBreadcrumbs.home'),
+      href: '/',
+    },
+    {
+      title: t('subscription.subscriptionBreadcrumbs.groups'),
+      href: '/groups',
+    },
+    {
+      title: groupId,
+      href: `/groups/${groupId}`,
+    },
+    {
+      title: topicName,
+      href: `/groups/${groupId}/topics/${topicName}`,
+    },
+  ];
+</script>
+
+<template>
+  <v-container class="d-flex flex-column topic-view__container">
+    <div class="d-flex justify-space-between align-center">
+      <v-breadcrumbs :items="breadcrumbsItems" density="compact" />
+    </div>
+    <loading-spinner v-if="topicIsLoading || ownerIsLoading" />
+    <console-alert
+      v-if="topicError"
+      :text="
+        t('topicView.errorMessage.topicFetchFailed', { topicName: topicName })
+      "
+      type="error"
+    />
+
+    <topic-header v-if="topic && owner" :topic="topic" :owner="owner" />
+
+    <div class="topic-view__upper_panel">
+      <metrics-list v-if="metrics" :metrics="metrics" />
+      <properties-list v-if="topic" :topic="topic" />
+    </div>
+
+    <schema-panel v-if="topic" :schema="topic.schema" />
+
+    <messages-preview v-if="messages" :messages="messages" />
+
+    <subscriptions-list v-if="subscriptions" :subscriptions="subscriptions" />
+  </v-container>
```

Comment:
What do You think about adding `Offline client` section? Same as in current console - [link](https://github.com/allegro/hermes/blob/6b5bb09bb21bf6ec870ba900f04c90f51449d134/hermes-console/static/partials/topic.html#L198-L209)

Reply:
Will do in another task